diff --git a/label/label.go b/label/label.go
index 137365a..8b85571 100644
--- a/label/label.go
+++ b/label/label.go
@@ -63,7 +63,7 @@ var NoLabel = Label{}
 
 var (
 	labelRepoRegexp = regexp.MustCompile(`^@$|^[A-Za-z][A-Za-z0-9_]*$`)
-	labelPkgRegexp  = regexp.MustCompile(`^[A-Za-z0-9/._-]*$`)
+	labelPkgRegexp  = regexp.MustCompile(`^[A-Za-z0-9/.@_-]*$`)
 	labelNameRegexp = regexp.MustCompile(`^[A-Za-z0-9_/.+=,@~-]*$`)
 )
 